gateway/src/main/resources/application.yaml

```
@@ -0,0 +1,11 @@
+spring:
+    application:
+        name: gateway
+server:
+    port: 8084
+eureka:
+    client:
+        service-url:
+            defaultZone: http://localhost:8761/eureka/
+        register-with-eureka: true
+        fetch-registry: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)